src/pages/Join/Join.js

```diff
@@ -100,46 +106,55 @@ function Join() {
         <div className="join-content-head">
           <h3 className="join-content-title">기본정보</h3>
           <p className="join-caution-text">
-            <span className="join-caution-symbol">*</span>표시는 반드시
-            입력하셔야 하는 항목입니다.
+            <span className="caution-symbol">*</span>표시는 반드시 입력하셔야
+            하는 항목입니다.
           </p>
         </div>
-        <form
-          className="join-form"
-          onChange={inputChange}
-          onSubmit={submitForm}
-        >
-          <div className="join-form-email">
-            <label>
-              <span className="join-caution-symbol">*</span>이메일
+        <form onChange={inputChange} onSubmit={submitForm}>
+          <div className="join-form">
+            <label className="form-label">
+              <span className="caution-symbol">*</span>이메일
             </label>
-            <input name="email" />
-            <button className="join-duplicate-btn" onClick={checkDuplicate}>
+            <input className="form-input-email input-focus" name="email" />
+            <button className="duplicate-btn" onClick={checkDuplicate}>
               중복확인
             </button>
           </div>
-          <div className="join-form-password">
-            <label>
-              <span className="join-caution-symbol">*</span>비밀번호
+          <div className="join-form join-form-pw">
+            <label className="form-label">
+              <span className="caution-symbol">*</span>비밀번호
             </label>
-            <input
-              name="password"
-              placeholder="영문 대소문자와 특수문자 포함, 8자리 이상"
-            />
+            <div className="form-password-wrap">
+              <input
+                className="form-input-long input-focus"
+                name="password"
+                placeholder="영문 대소문자와 특수문자 포함, 8자리 이상"
+              />
+              <span
+                className="join-valid-text"
+                style={
+                  password.length > 0
+                    ? handleValidateDisplay
+                    : { display: 'none' }
+                }
+              >
+                * 비밀번호 형식을 다시 확인해주세요.
+              </span>
```

Comment:
위 리뷰 참고하셔서 다음처럼 조건부 렌더링 하시면 됩니다.
```suggestion
              {isPwValid && (
                <span className="join-valid-text">
                  * 비밀번호 형식을 다시 확인해주세요.
                </span>
```